### DISCUSSION TITLE
nightly/aspnet:6.0-jammy-chiseled ICU support

### DISCUSSION THREAD

**Discussion by artem-voropaev** [dotnet/dotnet-docker#4714] [Q&A] [answered]
Hello team!
I use preview mcr.microsoft.com/dotnet/nightly/aspnet:6.0-jammy-chiseled docker image and I notice that it doesn't have localization support by default. 
I've got an exception on startup 

 Process terminated. Couldn't find a valid ICU package installed on the system. Please install libicu using your package manager and try again. Alternatively you can set the configuration flag System.Globalization.Invariant to true if you want to run with no globalization support. Please see https://aka.ms/dotnet-missing-libicu for more information.

Is it correct and I should install ICU manually?

**Comment by mthalman** [comment, marked as answer]
That's correct. ICU support is excluded by default because of its size. See the [globalization doc](https://github.com/dotnet/dotnet-docker/blob/main/samples/enable-globalization.md#icu) for more info.

**Reply by artem-voropaev** [reply]
Thanks, I found the way to include ICU in Chiseled image
